tests/ccgbugs/t13902.nim

```diff
@@ -0,0 +1,16 @@
+discard """
+  cmd: "nim c -r --skipParentCfg --skipUserCfg --cpu:i386 --passC:-m32 --passL:-m32 --stacktrace:off --hints:off $file"
+  action: run
+"""
+
+#issue #13902
+when defined(linux):
+  block:
+    type Slot = distinct uint64
+    var s = Slot(1)
+    proc `$`(x: Slot): string {.borrow.}
+    proc `+=`(x: var Slot, y: uint64) {.borrow.}
+    echo "s = ", s
```

Comment:
I guess you can remove the two echos here.

Reply:
i could; but note that the test was failing with either 0 echo or 2 echo but not with either echo (go figure...). I think it's fine like that because of this

Reply:
OK

Reply:
ended up removing both echos